CMakeLists.txt

```diff
@@ -102,22 +106,44 @@ if(TARGET_PLATFORM STREQUAL "linux")
 endif()
 
 # Default values
-set(BEEROCKS_WLAN1_IFACE "wlan0")
-set(BEEROCKS_WLAN2_IFACE "wlan2")
-set(BEEROCKS_WLAN3_IFACE "wlan4")
-set(BEEROCKS_BRIDGE_IFACE "br-lan")
+if(NOT DEFINED BEEROCKS_WLAN1_IFACE)
+    set(BEEROCKS_WLAN1_IFACE "wlan0")
+endif()
+if(NOT DEFINED BEEROCKS_WLAN2_IFACE)
+    set(BEEROCKS_WLAN2_IFACE "wlan2")
+endif()
+if(NOT DEFINED BEEROCKS_WLAN3_IFACE)
+    set(BEEROCKS_WLAN3_IFACE "wlan4")
+endif()
+if(NOT DEFINED BEEROCKS_BRIDGE_IFACE)
+    set(BEEROCKS_BRIDGE_IFACE "br-lan")
+endif()
 if (NOT DEFINED BEEROCKS_BH_WIRE_IFACE)
     set(BEEROCKS_BH_WIRE_IFACE "eth1")
 endif()
-set(BEEROCKS_REPEATER_MODE 0)
+if (NOT DEFINED BEEROCKS_REPEATER_MODE)
+    set(BEEROCKS_REPEATER_MODE 0)
+endif()
```

Comment:
Do you really need to change this for the RDKB build? Please only make this modification for stuff that really needs to be changed from the outside...

Reply:
No
Reverted changes for it